config/constants.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  connectDB: {
+    success: 'db is connected successfully',
+    failure: 'db could not be connected',
+  },
+  labels: {
+    githubPassportStrategy: {
+      githubProfile: {
+        failure: 'Github profile.id is not found!',
+      },
+      findingUser: {
+        error: 'Error while finding the user if its already existed in db (github)',
+        found: 'User is existed in db (github)',
```

Comment:
```suggestion
        error:
        found:
```
There should be only two keys success and failure. Please update this file accordingly.

Reply:
Done

Reply:
Not addressed

Reply:
Done